README.md

```diff
@@ -1,27 +1,22 @@
-# status page for conda-forge
+# Status page for conda-forge
 
-the [status page](https://conda-forge.org/status/) for conda-forge
+This repository serves as the incident source for the [status page](https://conda-forge.org/status/) in conda-forge.org.
 
-## usage
+## Usage
 
 To make an incident, open an issue and label it with one of
 
   - investigating
   - degraded performance
   - major outage
 
-Then GitHub Actions will update the page with the issue title and the
-text of the issue body.
+The [status page](https://conda-forge.org/status/) will automatically fetch the issues
+and render them as incidents.
 
-When the incident is resolved, close the issue and the status page will
-return to normal.
```

Comment:
Let's keep this unless you think it is too obvious?

Reply:
Agreed